django/chat/static/chat/js/scripts.js

```diff
@@ -95,6 +95,18 @@ function scrollToBottom(smooth = true, force = false) {
   messagesContainer.scrollTop = hashContainer ? messagesContainer.scrollTop + offset : messagesContainer.scrollHeight;
 }
 
+function scrollToListItem() {
+  setTimeout(() => {
+    const currentChat = document.querySelector('.chat-list-item.current');
+    if (currentChat) {
+      currentChat.scrollIntoView({
+        behavior: 'smooth',
```

Comment:
Personally I'd prefer it didn't smooth scroll (a lot going on with 2 panes scrolling at once on load)